src/server/server-interface.ts

```diff
@@ -1,6 +1,8 @@
 import { GameState } from '../core/game';
 
+export const GAME_START_TIME = 5000;
```

Comment:
Would prefer something like `GAME_LOBBY_COUNTDOWN`, something specifying that its jsut for the lobby.